frontend/sass/styles.scss

```diff
@@ -185,6 +185,9 @@ input:focus + .jf-checkbox-symbol {
 @mixin button-text-wrap() {
     white-space: normal;
     height: auto;
+    @media screen and (max-width: 400px) {
```

Comment:
hmm i wonder if you want to use `$jf-supertiny` here instead of 400px?  `$jf-supertiny` is 480px but it seems good to use as few breakpoints as possible, if possble...

Reply:
sounds good to me! Unfortunately, this means that all of our buttons on mobile will have a slightly shorter line height regardless... adding a comment now